packages/ui/components/editor/Editor.tsx

```diff
@@ -63,7 +66,7 @@ export const Editor = (props: TextEditorProps) => {
             excludedToolbarItems={props.excludedToolbarItems}
             variables={props.variables}
           />
-          <div className="editor-inner">
+          <div className={`editor-inner ${props.height}`}>
```

Comment:
Please pass this as inside a style tag - theres no way to know if the css for the entered prop is generated in our tailwind file (Or allow string to be a union type of things we know will be generated)

Reply:
good point